docs/getting-started/create-configuration.md

```diff
@@ -611,10 +611,10 @@ metadata:
     vpc: new
 spec:
   crossplane:
-    version: ">=v1.0.0-0"
+    version: ">=v1.2.0-0"
   dependsOn:
     - provider: crossplane/provider-aws
-      version: "v0.16.0"
+      version: "v0.18.1"
```

Comment:
Can you make sure to propagate any of the changes for the `Configuration` manifests to the appropriate `crossplane.yaml` in the `docs/snippets/package/...` directories? :)

Reply:
oh yes, indeed I missed a few things. This should be fixed now.